src/main/scala/devices/tilelink/Error.scala

```diff
@@ -18,7 +18,10 @@ class TLError(params: DevNullParams, beatBytes: Int = 4)(implicit p: Parameters)
     import TLPermissions._
 
     val (in, edge) = node.in(0)
-    val a = Queue(in.a, 1)
+    val a = if (params.minLatency > 0) {Queue(in.a, 1)} else in.a
+    require(params.minLatency  == 1 || params.minLatency == 0,
```

Comment:
@terpstra this is the actual change we want to make. And then correctly describe it.

Reply:
would it make you more comfortable if we call it something other than `minLatency` then describe it accordingly.

Reply:
This has been changed to be an explicit `buffer` parameter passed in, and then the TLError device communicates its `minLatency` to the parent class accordingly.